main.nf

```diff
@@ -394,6 +394,81 @@ process sentieon_qc {
 	"""
 }
 
+process sentieon_mitochondrial_qc {
+
+    // Fetch mitochondrial coverage statistics
+    // Calculate mean_coverage and pct_above_500x
+    
+    cpus 52
+    memory '30 GB'
+	tag "$id"
+	cache 'deep'
+	time '2h'
+	scratch true
+	stageInMode 'copy'
+	stageOutMode 'copy'
+	container = "/fs1/resources/containers/sentieon_202112.sif"
+
+	when:
+		params.antype == "wgs"
+    
+	input:
+        set group, id, file(bam), file(bai) from qc_mito_bam.groupTuple()
+
+	output:
+    	group, id, file("${id}_mito_coverage.tsv") into <<what?>>
+		
+	"""
+	sentieon driver \\
+		-r $genome_file \\
+		-t ${task.cpus} \\
+		-i $bam \\
+		--algo CoverageMetrics --cov_thresh 500 mitochondrial_coverage_metrics.txt
+
+    head -1 mitochondrial_coverage_metrics.txt.sample_interval_summary > ${id}_mito_coverage.tsv
+    grep "^M" mitochondrial_coverage_metrics.txt.sample_interval_summary >> ${id}_mito_coverage.tsv
+
+	"""
+    
+}
+
+process add_mitochondrial_qc_to_qc_json {
+
+    cpus 52
+	tag "$id"
+    time "10m"
+	stageInMode 'copy'
+	stageOutMode 'copy'
+
+    when:
+        ...
+
+    input:
+        // id, group, mito_qc_file from sentieon_qc, mitochondrial_coverage_metrics.txt from <<sentieon_mitochondrial_qc process above>>
+        id, group, qc_json_file=file("${id}.QC") <<from where?>>
+
```

Comment:
In the input block above I want to take the mitoQC tsv as input, and also the QC json file from `sentieon_qc` as input and combine the both in the script block below.